streams/streams-scala/src/main/scala/org/apache/kafka/streams/scala/kstream/KTable.scala

```diff
@@ -22,15 +22,21 @@ package kstream
 
 import org.apache.kafka.common.utils.Bytes
 import org.apache.kafka.streams.kstream.{ValueJoiner, ValueTransformerWithKeySupplier, KTable => KTableJ}
-import org.apache.kafka.streams.scala.FunctionsCompatConversions._
-import org.apache.kafka.streams.scala.ImplicitConversions._
+import org.apache.kafka.streams.scala.FunctionsCompatConversions.{
+  FunctionFromFunction,
+  KeyValueMapperFromFunction,
+  MapperFromFunction,
+  PredicateFromFunction,
+  ValueMapperFromFunction,
+  ValueMapperWithKeyFromFunction,
```

Comment:
Extra `,` at the end of the line.  Maybe this is the issue with `Scala 2.11`? Line number matches up from the `stdout` error.